components/brave_shields/browser/ad_block_component_service_manager.cc

```diff
@@ -1,9 +1,9 @@
-/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
this is technically a "new" file, although according to GitHub it's similar enough to the previous one that it's renamed instead. I'll keep the older copyright here too.